CRM/Custom/Form/Field.php

```diff
@@ -748,6 +748,10 @@ public static function formRule($fields, $files, $self) {
                 $errors['option_value[' . $nextIndex . ']'] = ts('Duplicate Option values');
                 $_flagOption = 1;
               }
+              if (strpos($fields['option_value'][$start], ',') != FALSE) {
```

Comment:
This makes me wonder if we need to have a status check or something as well alerting if people have any already

Reply:
For existing option value(s) with comma will won't work for 'Autocomplete Select' custom field as a filter, doesn't affect another widget type as a filter like checkbox. But yeah it would be nice to inform user about this on status change sometihing like 'Use of "," is no longer supported in optionvalue for mult-select custom fields esp. for 'Autocomplete Select' widget'